libsoftwarecontainer/src/gateway/networkgateway.h

```diff
@@ -26,6 +26,65 @@
 #include "jansson.h"
 #include "netlink.h"
 
+
+/**
+ * @brief A rules entry for the treatment of packets.
+ */
+struct IPTableEntry
```

Comment:
Why a struct? Why not a class with methods that expose members rather than having a struct that exposes the variables directly?